flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/source/abilities/SupportsFilterPushDown.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.connector.source.abilities;
+
+import org.apache.flink.annotation.PublicEvolving;
+import org.apache.flink.table.connector.source.ScanTableSource;
+import org.apache.flink.table.expressions.ExpressionVisitor;
+import org.apache.flink.table.expressions.ResolvedExpression;
+
+import java.util.List;
+
+/**
+ * Enables to push down filters into a {@link ScanTableSource}.
+ *
+ * <p>By default, if this interface is not implemented, filters are applied in a subsequent operation
+ * after the source.
+ *
+ * <p>For efficiency, a source can push predicates further down in order to be close to the actual data
```

Comment:
nit: Can we stick to a single term? Either `predicates` or `filters`?

It's an advice from a course that I'm doing lately: https://developers.google.com/tech-writing/one/words#use_terms_consistently

Reply:
Nice source. I also added it to my reading list.